packages/hs-react-ui/src/components/Card/Card.tsx

```diff
@@ -51,11 +51,11 @@ export const CardContainer = styled(Div)`
 `;
 
 export const Header = styled(Div)`
-  ${() => {
+  ${({ hasBody, hasFooter }) => {
     const { colors } = useTheme();
 
     return `
-      padding: 1.5rem 1.5rem 0rem;
+      padding: 1.5rem 1.5rem ${!hasBody && !hasFooter ? '1.5rem' : '0rem'};
```

Comment:
Could you add some snapshots to cover the cases where it has a body or a footer when there's a header? Maybe some of the other permutations

Reply:
I add the some snapshot tests around this case. Also, I updated this statement to make it more clear.